DataFormats/L1TMuonPhase2/src/classes_def.xml

```diff
@@ -8,8 +8,8 @@
  <class name="edm::Wrapper<std::vector<l1t::MuonStub > >"/>
  <class name="edm::Ref<l1t::MuonStubCollection>" splitLevel="0"/>
 
- <class name="l1t::TrackerMuon" ClassVersion="3">
-   <version ClassVersion="3" checksum="2397458791"/>
```

Comment:
Do not remove previous versions!
```
   <version ClassVersion="3" checksum="2397458791"/>
```
must be left below `ClassVersion="4"` here below.

A fix is in order as soon as next 2300 IB will become available

Reply:
My apologies, I should have caught this.

Reply:
Fix prepared in #41293: thank you @aloeliger !